app/models/miq_ae_method.rb

```diff
@@ -156,6 +158,10 @@ def self.find_best_match_by(user, relative_path)
 
   private
 
+  def embedded_methods_not_nil
+    errors.add(:embedded_methods, "is reserved") if embedded_methods.nil? || embedded_methods.include?(nil)
```

Comment:
What does the current validation print when it's violated?  I don't know what "is reserved" means.

Reply:
Agreed...this PR is simple enough, but I think we should make the error message match the previous value.